README.md

```diff
@@ -1,56 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Awesome Books
 
+In this project we are creating a live and interactive Todo List, with updating Todo.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://olamarx.github.io/Todo-List/dist)
```

Comment:
### Optional
- Please update your live link with the latest changes of your app for consistency.